package.json

```diff
@@ -23,12 +23,13 @@
   "scripts": {
     "eslint": "eslint 'src/**/*.ts'",
     "typecheck": "tsc -p tsconfig.json",
-    "build": "parcel build --out-file dist/contact-author.js src/contact-author.ts",
+    "build": "parcel build --out-file dist/contact-author.js src/activate.ts",
     "symlink-package": "mkdirp dist && lnfs ./package.json ./dist/package.json",
-    "serve": "yarn run symlink-package && parcel serve --no-hmr --out-file dist/contact-author.js src/contact-author.ts",
+    "serve": "yarn run symlink-package && parcel serve --no-hmr --out-file dist/contact-author.js src/activate.ts",
```

Comment:
Keeping the extension named after the extension name makes it easier to identify in Chrome dev tools (stack traces, sources panel etc)

Reply:
Thanks for pointing that out -- I didn't think of that reason for keeping the file name.